.github/workflows/main.yml

```diff
@@ -92,11 +92,11 @@ jobs:
           projectToken: ${{ secrets.CHROMATIC_PROJECT_TOKEN }}
           onlyChanged: "!startsWith(github.ref, 'refs/heads/main')" # Use TurboSnap for PR builds
           externals: |
```

Comment:
The menu storybook test diffs look pretty broken in both consecutive builds
![image](https://github.com/ni/nimble/assets/1588923/8a548699-32b8-4976-83dd-e3428e7c59b8)

Reply:
Anchored region was not being registered. Fixing in #2051.